cli/joy.js

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+'use strict';
+const fs = require('fs');
+const prog = require('caporal');
+const path = require('path');
+const pa11y = require('pa11y');
+const Docker = require('./plugins/docker');
+const Test = require('./plugins/test');
+const util = require('util');
+
+class Joy {
+  constructor() {
+    this.env = process.env;
+    this.prog = prog;
+    this.prog.version('1.0.0');
+  }
+
+  addCommand(commandObj) {
```

Comment:
This could use a function header comment block.

Reply:
Will do